change/react-native-platform-override-77ef9896-f6ff-43b2-a7ba-be92a933e696.json

```diff
@@ -1,7 +0,0 @@
-{
-  "type": "patch",
```

Comment:
why is this here?

Reply:
yarn change complained that it's a private package and it shouldn't have a change file there.